src/Orchard.Web/Web.config

```diff
@@ -225,7 +225,11 @@
             <dependentAssembly>
                 <assemblyIdentity name="Iesi.Collections" publicKeyToken="aa95f207798dfdb4" culture="neutral" />
                 <bindingRedirect oldVersion="0.0.0.0-4.0.0.0" newVersion="4.0.0.0" />
-            </dependentAssembly>            
+            </dependentAssembly>
+            <dependentAssembly>
```

Comment:
Sorry, it got added automatically when adding the YamlDotNet Nuget package. Should I remove it?